.github/ISSUE_TEMPLATE/failing-test.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Failing Test
+about: Report test failures
+
+---
+
+<!-- Please only use this template for submitting reports about failing tests -->
```

Comment:
Same here.. can we call out this should only be for Kubernetes CI test failures, not other kinds of possible failing tests